pep-0001.txt

```diff
@@ -148,10 +148,26 @@ The standard PEP workflow is:
 * You, the PEP author, fork the `PEP repository`_, and create a file named
   ``pep-9999.rst`` that contains your new PEP.  Use "9999" as your draft PEP
   number.
+
 * Push this to your GitHub fork and submit a pull request.
-* The PEP editors review your PR for structure, formatting, and other errors.
+
+* The PEP editors review your PR for structure, formatting, and other
+  errors.  Approval criteria are:
+
+  * It sound and complete.  The ideas must make technical sense.  The
+    editors do not consider whether they seem likely to be accepted.
+  * The title accurately describes the content.
+  * The PEP's language (spelling, grammar, sentence structure, etc.)
+    and markup (for reST PEPs) are correct, and code style (examples
+    should match PEP 8 & PEP 7) is good.  The PEP editors will edit
+    language and code style, but they can't approve it if they can't
+    parse it.  Do your best.
```

Comment:
That's a high bar, and honestly most of the time the PEP editors don't have time to check this carefully. Basically if the formatting test on Travis CI passes a PEP editor may well merge the PR without reading it (unless the first few paragraphs sound like the PEP author doesn't know what they're doing).

Reply:
Again, the requirements are a quote from "Editor Workflow".  New PR with a separate commit changing both this section and "Editor Workflow".

If I understand your "high bar" correctly, I propose softening it to something like "While the expectation is that most technical problems will be ironed out during the discussion, editors may request revisions for language and markup, and code style (PEPs 7/8).  Most important is that your PEP pass the formatting test in Travis CI.  (The result is visible in your PR, or you can use rstpep2html locally to catch most reST problems.)"

I wonder how hard it would be to run style checkers on the code samples.